src/crypto/crypto_hash.cc

```diff
@@ -103,7 +109,8 @@ void Hash::New(const FunctionCallbackInfo<Value>& args) {
     xof_md_len = Just<unsigned int>(args[1].As<Uint32>()->Value());
   }
 
-  Hash* hash = new Hash(env, args.This());
+  Hash* hash = cppgc::MakeGarbageCollected<Hash>(
```

Comment:
Would you mind adding a comment to here why we don't call new Hash but prefer the recommended change to here.